### PR TITLE
Python script for Intrinsic Functions verify_args()

### PR DESCRIPTION
**Pull request by Shaikh-Ubaid**
towards https://github.com/lfortran/lfortran/issues/3129.

### CODE REVIEW

**Comment by Shaikh-Ubaid** [comment]
We currently generate the `verify_args()` for about `20` intrinsic functions. Next we need to support generating `create_IntrinsicFunction()` via the script.

**Comment by Shaikh-Ubaid** [comment]
Ready.

**Review by certik — APPROVED** [review]
I think this is good. It already removes more lines than it adds. Good to merge, we can iterate on it with subsequent PRs.

Can you post an example of the generated file?

**Comment by Shaikh-Ubaid** [comment]
>Can you post an example of the generated file?

Sure, please find https://gist.github.com/Shaikh-Ubaid/aaafbad1865b9548f3af7b760bccd8c1.

**Comment by certik** [comment]
Thanks. I would remove the `        if (x.n_args == 2)  {` in there, since what if the function in ASR has a different number of arguments? It seems that aaaften

**Comment by Shaikh-Ubaid** [comment]
>what if the function in ASR has a different number of arguments?

` if (x.n_args == 2)` helps in the case of different number of arguments. `Anint` has different number of arguments. Please see https://gist.github.com/Shaikh-Ubaid/aaafbad1865b9548f3af7b760bccd8c1#file-intrinsic_function_registry_util-h-L205-L219.

Please share what we should do.

**Comment by certik** [comment]
I just created an issue https://github.com/lfortran/lfortran/issues/3159 that explains what the problem is and how to fix it.

**Comment by HarshitaKalani** [comment]
This looks great and promising! :raised_hands:

File: src/libasr/intrinsic_func_registry_util_gen.py [hunk 1]
```diff
@@ -0,0 +1,233 @@
...
+            "args": [("real", "int")]
+        }
+    ],
+    "Anint": [
+        {
+            "args": [("real",)]
+        },
+        {
+            "args": [("real", "int")]
+        }
```
Comment (certik) [inline src/libasr/intrinsic_func_registry_util_gen.py:94]: This can be removed

File: src/libasr/intrinsic_func_registry_util_gen.py [hunk 2]
```diff
@@ -0,0 +1,233 @@
...
+            "args": [("int",), ("real",)],
+        },
+    ],
+    "Aint": [
+        {
+            "args": [("real",)]
+        },
+        {
+            "args": [("real", "int")]
+        }
```
Comment (certik) [inline src/libasr/intrinsic_func_registry_util_gen.py:86]: Remove this one

**Pull request merged**